index.js

```diff
@@ -1898,7 +1898,7 @@ export async function createJavaBom(path, options) {
   // NOTE: This can match BUILD files used by perl, so could lead to errors in some projects
   const bazelFiles = getAllFiles(
     path,
-    `${options.multiProject ? "**/" : ""}BUILD*`,
+    `${options.multiProject ? "**/" : ""}BUILD{,.bazel}`,
```

Comment:
Instead of looking for `BUILD` and `BUILD.bazel` files, it should instead:

1. `WORKSPACE` - since that defines a Bazel project (and also `WORKSPACE.bazel`), and only at the root directory of the project
2. `MODULE.bazel` - as an alternative to `WORKSPACE` in newer versions of Bazel

As the error suggests we can only run Bazel build commands from "below a directory having a WORKSPACE file" so even just looking for `BUILD` and `BUILD.bazel` will still result in failures.

Reply:
so
```
`(WORKSPACE{,.bazel}|${options.multiProject ? "**/" : ""}BUILD{,.bazel})`
```
would be the pattern?

or can't it be covered with the pattern at all as it's a conditional thing, depending on a WORKSPACE file to be present or not?

Reply:
I have some bazel projects without any workspace files. Not sure since when workspace was introduced

Reply:
It needs to find a `WORKSPACE` or a `MODULE.bazel` file at the root of the project, so something like
`(WORKSPACE{,.bazel}|MODULE.bazel)`

Alternatively you can run `bazel info` and look for the message 
```ERROR: The 'info' command is only supported from within a workspace (below a directory having a WORKSPACE file).```

Reply:
Let me try this in a new PR